packages/event-bus-redis/src/services/event-bus-redis.ts

```diff
@@ -71,26 +47,30 @@ export default class RedisEventBusService extends AbstractEventBusModuleService
     this.config_ = configModule
     this.logger_ = logger
 
-    this.connect()
+    const connection = this.connect()
+
+    // Register our worker to handle emit calls
+    new Worker(this.moduleOptions_.queueName ?? "events-queue", this.worker_, {
+      connection,
+      prefix: `${this.constructor.name}`,
+      ...(this.moduleOptions_.workerOptions ?? {}),
+    })
   }
 
-  connect(): void {
-    // Required config
-    // See: https://github.com/OptimalBits/bull/blob/develop/CHANGELOG.md#breaking-changes
+  connect(): ConnectionOptions {
     const connection = new Redis(this.moduleOptions_?.redisUrl!, {
+      // Required config. See: https://github.com/OptimalBits/bull/blob/develop/CHANGELOG.md#breaking-changes
       maxRetriesPerRequest: null,
+      ...(this.moduleOptions_.redisOptions ?? {}),
     })
 
     this.queue_ = new Queue(this.moduleOptions_.queueName ?? `events-queue`, {
       connection,
-      prefix: this.moduleOptions_.queuePrefix ?? `${this.constructor.name}`,
+      prefix: `${this.constructor.name}`,
```

Comment:
question: Any reason why it is not configurable anymore?

Reply:
You can still configure it through `queueOptions` as they are spread after the default prefix is set

Reply:
yep sorry, missed it